dev/com.ibm.ws.org.apache.cxf.cxf.rt.frontend.jaxws.3.2/src/org/apache/cxf/jaxws/ServiceImpl.java

```diff
@@ -540,12 +629,20 @@ private EndpointInfo createEndpointInfo(AbstractServiceFactoryBean serviceFactor
             //ignore
         }
         DestinationFactory df = dfm.getDestinationFactoryForUri(address);
-
+        if (isFinestEnabled) {   // Liberty Change issue #26529
+            LOG.finest("DestinationFactory that is obtained from DestinationFactoryManager: " + df + ", with address: " + address);
```

Comment:
You probably need "df.getClass().getCanonicalName()" here, just df may not log anything useful.